src/graphql/index.ts

```diff
@@ -23,6 +23,7 @@ export default graphqlHTTP((req) => ({
       cms: createDataLoader(''),
       data: createDataLoader(req.headers.authorization || ''),
       datasets: createDataLoader(''),
+      openAPI: createDataLoader(''),
```

Comment:
A whole new dataloader? How come this was missing?

Reply:
Well we used to use the `datasets`  loader to get results from openAPI in the dataset resolver. However, the results from openAPI are also used in filters resolver (thus using the datasets loader), so I thought it'd make more sense to create a separate loader for this.

Reply:
Cool, sounds good to me :+1: